qa/tasks/mgr/dashboard/test_pool.py

```diff
@@ -71,49 +112,27 @@ def _get_pool(self, pool_name):
         self.assertSchemaBody(self.pool_schema)
         return pool
 
-    def _check_pool_property(self, prop, value, pool):
-        if prop == 'pool_type':
-            self.assertEqual(pool['type'], value)
-        elif prop == 'size':
-            self.assertEqual(pool[prop], int(value), '{}: {} != {}'.format(prop, pool[prop], value))
-        elif prop == 'pg_num':
-            self._check_pg_num(value, pool)
-        elif prop == 'application_metadata':
-            self.assertIsInstance(pool[prop], list)
-            self.assertEqual(pool[prop], value)
-        elif prop == 'pool':
-            self.assertEqual(pool['pool_name'], value)
-        elif prop.startswith('compression'):
-            if value is not None:
-                if prop.endswith('size'):
-                    value = int(value)
-                elif prop.endswith('ratio'):
-                    value = float(value)
-            self.assertEqual(pool['options'].get(prop), value)
-        else:
-            self.assertEqual(pool[prop], value, '{}: {} != {}'.format(prop, pool[prop], value))
-
     def _check_pg_num(self, value, pool):
-        # If both properties have not the same value, the cluster goes into a warning state,
-        # which will only happen during a pg update on a existing pool.
-        # The test that does that is currently commented out because
-        # our QA systems can't deal with the change.
-        # Feel free to test it locally.
-        prop = 'pg_num'
+        """
+        If both properties have not the same value, the cluster goes into a warning state, which
+        will only happen during a pg update on an existing pool. The test that does that is
+        currently commented out because our QA systems can't deal with the change. Feel free to test
+        it locally.
+        """
         pgp_prop = 'pg_placement_num'
-        health = lambda: self._get('/api/health/minimal')['health']['status'] == 'HEALTH_OK'
-        t = 0;
-        while (int(value) != pool[pgp_prop] or not health()) and t < 180:
+        t = 0
+        while (int(value) != pool[pgp_prop] or self._get('/api/health/minimal')['health']['status']
+               != 'HEALTH_OK') and t < 180:
             time.sleep(2)
             t += 2
             pool = self._get_pool(pool['pool_name'])
-        for p in [prop, pgp_prop]:  # Should have the same values
+        for p in ['pg_num', pgp_prop]:  # Should have the same values
             self.assertEqual(pool[p], int(value), '{}: {} != {}'.format(p, pool[p], value))
 
     @classmethod
     def tearDownClass(cls):
         super(PoolTest, cls).tearDownClass()
-        for name in ['dashboard_pool1', 'dashboard_pool2', 'dashboard_pool3', 'dashboard_pool_update1']:
+        for name in ['dashboard_pool1', 'dashboard_pool2', 'dashboard_pool3', 'dashboard_pool_update']:
```

Comment:
I'm wondering if pool `dashboard_pool_update` needs to be deleted here. Where is it created?

Reply:
Thanks! It was a remnant of the previous implementation.